library/data/recipe/src/main/resources/data/quilt/tags/items/potions.json

```diff
@@ -0,0 +1,9 @@
+{
+  "replace": false,
+  "values": [
```

Comment:
This tag name sounds way too global.

I rather would have a  `quilt:brewing_stand/recipients.json` and include `c:potions` in it.